stdlib/public/Cxx/UnsafeCxxIterators.swift

```diff
@@ -18,12 +18,13 @@
 ///
 /// - SeeAlso: https://en.cppreference.com/w/cpp/named_req/InputIterator
 public protocol UnsafeCxxInputIterator: Equatable {
-  associatedtype Pointee
+  associatedtype Pointee: ~Copyable
 
   /// Returns the unwrapped result of C++ `operator*()`.
   ///
   /// Generally, Swift creates this property automatically for C++ types that
   /// define `operator*()`.
+  @_borrowed
```

Comment:
what the heck is this thing? I assume it means this getter borrows?

Reply:
Yeah! It's an old attribute that changes the property requirement to allow noncopyable use. For instance, we use it on Collection's subscript

Reply:
We need to have a non-underscored way to spell `@_borrowed` for a world where protocols fully understand non-copyable types.